sdk/src/Services/S3/Custom/Transfer/TransferUtilityDownloadDirectoryResponse.cs

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
this is placeholder for now. will have to populate it when we create the new DownloadDirectoryWithResponse api and similar for others